src/config/external-applications.md

```diff
@@ -0,0 +1,83 @@
+# External Applications
+
+## Programming Languages
+
+> The Void repositories have a number of Python and Lua packages. If possible,
+> install packages from the Void repositories or consider packaging the library
+> or application you need. Packaging your application allows for easier system
+> maintenance and can benefit other Void Linux users, so consider making a pull
+> request for it. The contribution instructions can be found
+> [here](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
+
+To keep packages smaller, Void has separate `devel` packages for header files
+and development tools. If you install a library or application via a language's
+package manager (i.e. `pip`, `gem`) or compile one from source, you may need to
+install the programming language's `-devel` package. This is specially relevant
+for `musl` libc users, due to pre-built binaries usually targeting `glibc`
+instead.
+
+| Language | Package Manager                | Void Package    |
+|----------|--------------------------------|-----------------|
+| Python3  | pip, anaconda, virtualenv, etc | `python3-devel` |
+| Python2  | pip, anaconda, virtualenv, etc | `python2-devel` |
+| Ruby     | gem                            | `ruby-devel`    |
+| lua      | luarocks                       | `lua-devel`     |
+
+## Restricted Packages
+
+Some packages have legal restrictions on their distribution (i.e. Discord), may
+be too large or have another condition that makes it difficult for the Void team
+to distribute. These packages are maintained but not distributed. As such, they
+must be built locally. For more information see the page on [restricted
+packages](../repositories/restricted.md).
+
+## Non-x86_64 Arch
+
+The Void build system runs on x86_64 computers, both for compiling and cross
```

Comment:
s/system/server/? Otherwise it might be read as implying that users must use x86_64 to build packages.